app/lib/efile/az/az321_calculator.rb

```diff
@@ -33,7 +33,8 @@ def calculate_line_4
 
   def calculate_line_5
     # Total contributions made or fees paid to QCOs during 2023
-    @intake.az321_contributions.sum(&:amount).round
```

Comment:
we do this in az322_contributions as well. should we update that as well?

Reply:
I'm not sure, but it'd be worth verifying with product. Thanks for catching